### ISSUE TITLE
Bug on install using  PiVPN easy-rsa

### ISSUE THREAD

**Issue by jpdias** [pivpn/pivpn#215]
# PiVPN Issue Template

## Console output of `curl install.pivpn.io | bash`
```
::: Using protocol: udp

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
chown: cannot access ‘/etc/openvpn/easy-rsa’: No such file or directory
mkdir: cannot create directory ‘/etc/openvpn/easy-rsa/pki’: No such file or directory
tee: /etc/openvpn/easy-rsa/vars: No such file or directory
bash: line 822: cd: /etc/openvpn/easy-rsa: No such file or directory
Synchronizing state for openvpn.service with sysvinit using update-rc.d...
Executing /usr/sbin/update-rc.d openvpn defaults
Executing /usr/sbin/update-rc.d openvpn enable

Rebooting system...
```

## Issue

I got an issue on the installation procedure. After some research I found out that the trouble was in the 
`Get the PiVPN easy-rsa` lines.  I don't know if anyone else experienced the same problem, but this works as a quick-fix: 

`sudo apt-get install openvpn easy-rsa`

`cp -r /usr/share/easy-rsa/ /etc/openvpn`

**Comment by redfast00** [comment]
It seems that the network connection might have been interrupted. Maybe we should check if download succeeded @0-kaladin?

**Comment by cfcolaco** [comment]
@redfast00 @0-kaladin this seems to be a one-time scenário, wanna keep this open for further improvements?

**Comment by 0-kaladin** [comment]
Nope. Future stuff will likely be half rewritten. This won't apply.

**Comment by cfcolaco** [comment]
Closing issue then.

**Comment by PBLCelt** [comment]
Ran into this same problem today.

**Comment by PGallagher69** [comment]
Just for reference, I ran into this problem today and solved it here... https://github.com/pivpn/pivpn/issues/674#issuecomment-481741747

**Comment by Giraffe1966** [comment]
The easy-rsa problem has been fixed (#716)

**Comment by SandcrawlerVault** [comment]
Ran into this issue today installing on an up to date DietPi, but using official PiVPN script. Fixed with workaround by jpdias.
